File: pachow.py
__author__ = 'sbanerjee'

import random

def main():
	hards = ['p', 'ch', 'k', 't', 'w', 'z', 'm']
	vowels = ['i', 'o', 'a']

	pachow = ''
	numSyllables = random.randint(1,4)
	for i in range(0, numSyllables):
		hard = random.choice(hards)
		vowel = random.choice(vowels)
		pachow += hard + vowel

	pachow += random.choice(hards)
	pachow += 'ow'
	print pachow

if __name__ == "__main__":
	main()

File: pachow.php
<?php

$hards = array(
		 'p',
		  'ch',
		   'k',
		    't',
		     'w',
		      'z',
		       'm',
	      );

$vowels = array(
		 'i',
		  'o',
		   'a',
	       );

$numSyllables = mt_rand(1,4);
$pachow = '';
for ($i = 0; $i < $numSyllables; ++$i) {
	 $hard = $hards[mt_rand(0, count($hards) - 1)];
	  $vowel = $vowels[mt_rand(0, count($vowels) - 1)];
	   $pachow .= $hard.$vowel;
}

echo $pachow.$hards[mt_rand(0, count($hards) - 1)].'ow';

